src/rtcModule/webrtc.cpp

```diff
@@ -853,6 +906,51 @@ void RtcModule::abortCallRetry(Id chatid)
     }
 }
 
+void RtcModule::refreshTurnServerIp()
+{
+    if (mIceServerProvider.empty())
+    {
+        return;
+    }
+
+    std::string url = mIceServerProvider[0]->url;
+    if (url.size())
+    {
+        size_t postInitialColon = url.find(":") + 1;
+        size_t postFinalColon = url.rfind(":");
+        std::string subString = url.substr(postInitialColon, postFinalColon - postInitialColon);
+        auto wptr = weakHandle();
+        int turnServerDnsRequestId = ++mTurnServerDnsRequest;
+        DnsRequest::getInstance()->wsResolveDNS(mKarereClient.websocketIO, subString.c_str(),
+                                                [wptr, this, subString, turnServerDnsRequestId]
+                                                (int statusDNS, const std::vector<std::string> &ipsv4, const std::vector<std::string> &ipsv6)
+        {
+            if (wptr.deleted() || turnServerDnsRequestId != mTurnServerDnsRequest)
+                return;
+
```

Comment:
I miss the management of `statusDNS < 0`, which indicates the DNS resolution has failed. In that case, I think the DNS records should not be updated.
Also, it would be good to add some error logging in this lambda function in order to ease debugging.

Reply:
I'll add it